src/app_service/service/community/dto/community.nim

```diff
@@ -57,6 +57,10 @@ type CommunityDto* = object
   muted*: bool
   pendingRequestsToJoin*: seq[CommunityMembershipRequestDto]
 
+type CommunitySettingsDto* = object
+  id*: string
+  historyArchiveSupportEnabled*: bool
```

Comment:
Do we plan to have a lot more community settings? If not really, I think it would be simpler and better to just put them in the CommunityDto as well as the status-go community return. Since it's a 1 to 1 relation ship, I don't think there is a good reason to separate them in different tables.

Reply:
There are more settings, just not exposed to the UI and I could imagine we'll add more settings in the future. But I'm happy to move them into the community object